src/MovieGrid/MovieGrid.js

```diff
@@ -34,10 +34,19 @@ class MovieGrid extends Component {
       ]
     }
   }
+
+  componentDidMount = () => {
```

Comment:
Do you want to try out some async await here? It might be something to come back to, but I've heard it's a good practice with fetch.